src/pudl/package_data/notebooks/how_to_access_pudl.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>I think it should be <code>pandas.Dataframe</code> and "database" vs. "databse" ??</p><p> </p><p>I would add something right off the bat that lists the different places where PUDL data is stored. It could also serve users to mention that there is a separate notebook for accessing the CEMS data.</p><p> </p><p>I think it'll be useful to establish the meaning of "PUDL" vs. the "PUDL database" and whether they are the same. Either that, or just stick to one.</p><p> </p><p>In the sentence that talks about database tables being more useful when combined, I think we should mention the output tables and that this is one method of viewing tables we are going to review in this notebook. I would then I add a list of the three methods for accessing them (maybe the headers from below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>